### PR TITLE
Feat: Playlist and Liked video pages

### PR DESCRIPTION
**Pull request by ramalingaa**
Added Playlist page
  User can now
    -Add videos to playlist
    - Add/remove playlist
Added Liked Page
  -User can now
      - View Liked videos
       -Remove liked videos

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
clean code and nice folder structure, looks good to merge !

File: src/App.css [hunk 1]
```diff
@@ -553,4 +553,85 @@ a {
...
+  align-items:center;
+}
+.playlist-cards-wrapper {
+  display:flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+.open-playlistBtn {
+  align-self: center;
+}
```
Comment (tejas-muthya37) [inline src/App.css:591]: Try using the naming convention whenever there are multiple words in your class name -> eg. btn-open-playlist

File: src/frontend/components/likedvideo/LikedVideo.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useVideoContext } from '../../context/index-context'
+import { SideMenuGuide, LikedVideoCard} from '../index-components'
+import { Link } from "react-router-dom"
+
+const LikedVideo = () => {
+  const { state } = useVideoContext()
+  const { likedData} = state
```
Comment (tejas-muthya37) [inline src/frontend/components/likedvideo/LikedVideo.jsx:8]: Try to destructure this in a single line if possible.

File: src/frontend/components/likedvideo/LikedVideo.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const LikedVideo = () => {
+  const { state } = useVideoContext()
+  const { likedData} = state  
+  return (
+    <div className="single-card-wrapper">
+      <SideMenuGuide />
+      <div className = "single-play-card">
+        {likedData.map((ele) => {
+            return <LikedVideoCard vInfo = {ele}/>
+        })}
```
Comment (tejas-muthya37) [inline src/frontend/components/likedvideo/LikedVideo.jsx:15]: Whenever you map on something, try to name the parameter as what each element is. Eg., Here, likedVideo.

**Review by tejas-muthya37 — COMMENTED** [review]
Liked your folder structure and use of async/await. Approving this PR.

**Review by tejas-muthya37 — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (ramalingaa) [reply src/App.css:591]: Sure

**Pull request merged**